docs/developer/bulks/bulk-attributes.mdx

```diff
@@ -0,0 +1,351 @@
+---
+title: Bulk Attributes
```

Comment:
I don't think "Bulk Attributes" is a good title as it's the operation that happens in bulk, not the attributes. "Bulk Attribute Updates," or "Bulk Attribute Import" would be better.

The same goes for the existing "Bulk Orders" document which to me sounds more like "placing large orders" than "creating many orders at once."